tutorials/snapcraft/create-your-first-snap.md

```diff
@@ -0,0 +1,726 @@
+  - We move one of the files to a different location.
+
+Luckily the second option is easy to implement and it’s nice being able to ship both. The `./configure` script of bash comes with an `--infodir` option, which will set the new location of the info directory. Let’s use `/var/bash/info`. All you need to do is make your gnu-bash definition look like this:
+
+
+
+```bash
+  gnu-bash:
+	source: http://ftp.gnu.org/gnu/bash/bash-4.3.tar.gz
+	plugin: autotools
+	configflags: ["--infodir=/var/bash/info"]
+```
+
+This is how you tell snapcraft to pass `--infodir=/var/bash/info` as an argument to `./configure` during the build.
+
+
+positive
+: Note: The `configflags` option is specific to the `autotools` plugin. Luckily `snapcraft` comes with a built-in help system. You can easily find out about all the options by running
+`snapcraft help sources` for source options,
+`snapcraft help <plugin-name>` for any given plugin,
+`snapcraft list-plugins` for a list of all available plugins.
+
+
+Now just run
+
+
+```bash
+~/hello$ snapcraft clean gnu-bash -s build
+Skipping cleaning priming area for gnu-bash (already clean)
+Skipping cleaning staging area for gnu-bash (already clean)
+Cleaning build for gnu-bash
+~/hello$ snapcraft prime
+[...]
+Skipping build gnu-hello (already ran)
+Staging gnu-bash
+Staging gnu-hello
+Priming gnu-bash
+Priming gnu-hello
+```
+
+Here you clean just the build step of the `gnu-bash` part, so the source does not need to be re-downloaded. Then the build is run again and it passes! Now let’s see if our new commands work fine.
+
+
+```bash
+~/hello$ sudo snap try --devmode prime
+
+hello 2.10 mounted from /home/daniel/hello/prime
+~/hello$ hello
+Hello, world!
+~/hello$ hello.bash
+bash-4.3$ env
+[ Environment variables list ]
+bash-4.3$ exit
+```
+
+
+positive
+: Note: You will see that the environment variables available from your snap are a little different from your user environment: some additional variables are added like **$SNAP_*** and some system environment variables have been altered to point to your snap directory, like **$PATH** or **$LD_LIBRARY_PATH**. Take the time to get familiar with them!
+
+
+Excellent work! You have it all nice and working!
+
+
+
+## Removing devmode
+Duration: 2:00
+
+
+positive
+: Lost or starting from here?
+Check or download [here][here2] to see what your current directory should look like.
+
+
+One last thing you might want to do before the snap is ready for wider consumption is to remove the `devmode` status.
+
+
+positive
+: Users of snaps using `devmode`, will need to pass `--devmode` during the installation, so they explicitly agree to trust you and your snap. Another benefit of removing devmode is that you will be able to ship your snap on the **stable** or **candidate** channels (you can only publish to the other channels, like **beta** or **edge** as your snap is less trusted) and users will be able to search for it using `snap find`.
+
```

Comment:
This is red/warning in the original:

![image](https://user-images.githubusercontent.com/7047542/27922084-cb66cb40-6271-11e7-9554-dd54727f116a.png)